manifests/role/ansible_master.pp

```diff
@@ -36,20 +36,22 @@
   $host_groups,
   $playbooks_source = 'https://github.com/openshift/openshift-ansible.git',
   $playbooks_version = 'master',
+  $ansible_inventory = '/etc/ansible/hosts',
+  $packages = [
```

Comment:
Adding more packages requires duplicating the existing values, and list updates become more difficult. I presume you want to define a list on your own. You can do that by adding a new boolean parameter named like `$install_required_packages`. That way external code can install the desired packages.